data-raw/area/area.R

```diff
@@ -12,14 +12,20 @@ lter$area_m2 = lter$area_ha * 1e4
 #merge to area
 area = rbind(nhd, lter[,c('site_id', 'area_m2')])
 
+# grab new data for footprint expansion
+new_area <- readRDS('data-raw/area/lakeattributes_area.rds')
```

Comment:
I wonder if this and similar appended files should be named something else, like `expanded_footprint` or `nd_sd_in`, etc... so that it is clear that it is in addition to the lakeattributes original area

Reply:
Yeah I debated on this for a while. I liked this as something we could overwrite, say, if we found depth data for a few lakes and updated the `lake-temp-model-prep` pipeline and could easily copy and paste over. Thoughts?

Reply:
Ahh yeah that makes sense, good thinking. Leaving as you have it seems like a good option then.